presentation/components/src/test/instance-filter-builder/Common.ts

```diff
@@ -0,0 +1,36 @@
+/*---------------------------------------------------------------------------------------------
+* Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+* See LICENSE.md in the project root for license terms and full copyright notice.
+*--------------------------------------------------------------------------------------------*/
+
+// need to stub global RAF and CAF for 'react-select' component
+export function stubRaf() {
```

Comment:
Please lean towards not using abbreviations. If you must, then explain each abbreviation once in a doc comment. The line above this function is not a doc comment because it does not start with '/**`.